cms_form/models/ir_qweb.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+import ast
+
+from odoo import models
+
+# TODO: get rid of this override as soon as this PR is merged https://github.com/OCA/OCB/pull/654  # noqa
```

Comment:
When this is merged, will we have a conflict?

Reply:
why? we are overriding the whole method in ir_qweb model (which is the only nice way to do it actually).

Reply:
Just asking to be sure.

Anyway I feel we'll have to keep the patch here if we target upstream odoo too, right? So it would be good to replace this with `# TODO Remove file in v11` IMHO.

Reply:
Ok, doing it. Then I'm not even sure if we should keep the backport PR :thinking:

Reply:
If you're only using it for this PR, I'd say it's up to you... But AFAIK supporting upstream odoo is a requirement in OCA 😕